packages/connectivity/src/scp-cf/destination/destination-service.ts

```diff
@@ -164,29 +167,13 @@ export async function fetchDestination(
 async function fetchDestinationByTokens(
   destinationServiceUri: string,
   tokens: AuthAndExchangeTokens,
-  options: DestinationFetchOptions
+  options: DestionServiceOptions
 ): Promise<Destination> {
   const targetUri = `${destinationServiceUri.replace(
     /\/$/,
     ''
   )}/destination-configuration/v1/destinations/${options.destinationName}`;
 
-  if (options?.useCache) {
```

Comment:
Maybe I miss something.
Could you please point out the potential tests about `useCache` changes.

Also, I see the description below, does the test also cover this? 

> If a IsolationStrategy is given to the getDestination() this enables the cache. No need to provided useCache: true

Reply:
The tests are under `caching options` in the `destination-cache.spec.ts`